0066-swiftui-and-state-management-pt2/StateManagement.playground/Contents.swift

```diff
@@ -187,16 +186,15 @@ struct IsPrimeModalView: View {
 }
 
 struct FavoritePrimesView: View {
-  @ObjectBinding var state: AppState
+  @ObservedObject var state: AppState
 
   var body: some View {
     List {
-      ForEach(self.state.favoritePrimes) { prime in
+      ForEach(self.state.favoritePrimes, id: \.self) { prime in
```

Comment:
this also seems to be required to help the compiler understand what is going on.

here `favoritePrimes` is an array of ints, but apparently the `ID` of a `ForEach` only needs to be `Hashable`, not `Identifiable`.